Decisions.Equifax/ConsumerCreditReport/Response/ConsumerCreditReportResponse.cs

```diff
@@ -20,7 +20,7 @@ public partial class ConsumerCreditReportResponse
     [Writable]
```

Comment:
We didn't have to add the custom converter anywhere for the Limited Consumer Report Response?

Reply:
not at all